en/docs/setup/installing-on-windows.md

```diff
@@ -50,7 +49,7 @@ by sorting by date.
 You set up `JAVA_HOME` using the System Properties, as described below.
 Alternatively, if you just want to set JAVA_HOME temporarily for the
 current command prompt window, [set it at the command
-prompt](#setting-java_home-temporarily-using-the-windows-command-prompt-(cmd)).
+prompt](#java-home).
```

Comment:
I believe this should link to #setting-java_home-temporarily-using-the-windows-command-prompt-cmd

Reply:
Link doesn't work properly probably because of the underscore. So added an anchor link right above the header and that works.